### ISSUE TITLE
Do current GHC versions build on 10.6.8 (i386/x86_64, of course)?

### ISSUE THREAD

**Issue by barracuda156** [kencu/ghc-for-older-darwin-systems#3]
@kencu Ken, it has been a while since I touched this topic, but I wanna return to it. Since I will need to restore powerpc code into GHC, it matters which version to work with. Therefore the question: is it known that later versions (post-9.4.x) are broken on 10.6.8 or you just did not make new releases for those? 
And just in case, did you ever try building this on 10.5.8 Intel?

**Comment by kencu** [comment]
The ghc build system underwent a significant change to haddock after this. I haven’t tried anything newer since the change. 

I did try building on 10.5.8 Intel, with some trials and tribulations that stopped me from working on it further at the time.

**Comment by kencu** [comment]
Now that I think about it, the issue with 10.5 mIght have been the 32 bit code.

**Comment by barracuda156** [comment]
Thanks. I took a quick look at changes past 9.2.x (which was the last version I started to restore ppc code into), and more stuff could have been broken since then.

GHC is an odd case personally: I have no software for which I may need Haskell (is anything even been written in it besides NASA software?), but I have an interest in category theory, and had an idea that Haskell could be helpful in that context.
